### PR TITLE
Enable redis-cluster backend to consume from multiple redis clusters

### PR DESCRIPTION
**Pull request by dlunch**
Enable kombu redis-cluster transport to consume from alt redis-cluster specified by alt parameter in query string

### CODE REVIEW

File: kombu/transport/redis_cluster.py [hunk 1]
```diff
@@ -511,6 +522,7 @@ class Transport(RedisTransport):
 
     driver_type = 'redis-cluster'
     driver_name = driver_type
+    can_parse_url = True
```
Comment (neoatlan) [inline kombu/transport/redis_cluster.py:525]: Is this property being used?

File: kombu/transport/redis_cluster.py [hunk 2]
```diff
@@ -375,62 +378,70 @@ def conn_or_acquire(self, client=None):
...
-        password = conninfo.password
-        transport = self.connection.client.transport_cls
-        ssl = transport == 'rediss-cluster'
+        parsed = parse_url(conninfo.hostname)
+        ssl = parsed['transport'] == 'rediss-cluster'
+
+        connection = RedisClusterConnection.get_consumer_connection(parsed['hostname'], parsed['port'], parsed['password'], ssl)
 
-        return RedisClusterConnection.get_consumer_connection(hostname, port, password, ssl)
+        # Additional redis cluster
```
Comment (neoatlan) [inline kombu/transport/redis_cluster.py:391]: Can alt be only one cluster?

File: kombu/transport/redis_cluster.py (hunk 1 above)
Comment (dlunch) [reply kombu/transport/redis_cluster.py:525]: Yes, it's used at https://github.com/sendbird/kombu/blob/75027490c71b83342f92b5293461c679233dc25e/kombu/connection.py#L675

File: kombu/transport/redis_cluster.py (hunk 2 above)
Comment (dlunch) [reply kombu/transport/redis_cluster.py:391]: Yes, it's only one alt for now

File: kombu/transport/redis_cluster.py (hunk 1 above)
Comment (neoatlan) [reply kombu/transport/redis_cluster.py:525]: Ah I see. Thanks

File: kombu/transport/redis_cluster.py (hunk 2 above)
Comment (neoatlan) [reply kombu/transport/redis_cluster.py:391]: Aha. Thanks

**Review by neoatlan — APPROVED** [review]

**Pull request merged**